envergo/moulinette/models.py

```diff
@@ -828,6 +838,10 @@ class ConfigHaie(ConfigBase):
 
     contacts_and_links = models.TextField("Champ html d’information", blank=True)
 
+    natura2000_coordinators_list_url = models.URLField(
+        "Liste des animateurs Natura2000", blank=True
```

Comment:
Pour info il est proposé une modif du label pour ce champs dans [le ticket](https://trello.com/c/u2teypIN/1543-it%C3%A9rations-sur-ladmin-r%C3%A9glementation-et-config) que je suis en train de traiter.

```suggestion
        "URL liste des animateurs Natura 2000", blank=True
```